app/views/shared/_address_fields.html.slim

```diff
@@ -0,0 +1,9 @@
+.nested-fields
+  = f.input :name
```

Comment:
I think asking for name w/ address is redundant since we're already asking for the name just before it in every instance I see -  patient, caretaker...